hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/generator/CounterRow.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.generator;
+
+import com.hazelcast.sql.impl.row.Row;
+
+import java.util.Objects;
+
+/**
+ * Row with single integer field
+ */
+class CounterRow implements Row {
```

Comment:
Why "counter" in the name? It's just a value here, not a counter.

Also, in contract to the long variant, this should contain `Int` in the name:
```suggestion
class SingleIntRow implements Row {
```

Reply:
It is only internal class, used as counter, but I like your name better

Reply:
Even the name of a local variable matters, it should ideally say what the value is.